README.md

```diff
@@ -99,6 +99,14 @@ Type: `string`
 
 The following input variables are optional (have default values):
 
+### azure\_velero\_plugin\_version
+
+Description: Version of the azure velero plugin to use
+
+Type: `string`
+
+Default: `"v1.3.0"`
+
 ### excluded\_namespaces
```

Comment:
well..... i might be very picky, but if we say "include", then it should also be "exclude", not "excluded"

Reply:
No, you're right.